snapshots/jest.e2e.config.ts

```diff
@@ -0,0 +1,11 @@
+process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD = 'true';
```

Comment:
If the e2e-snapshots script already sets PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true, do you need this set here too?

Reply:
Nit really they actually do the same thing. I will remove it from this file